src/components/bodyImageThumbnail.tsx

```diff
@@ -0,0 +1,38 @@
+// ----- Imports ----- //
+
+import React, { FC, ReactNode } from 'react';
+import { css } from '@emotion/core';
+import { breakpoints } from '@guardian/src-foundations';
+
+import Image, { Props as ImageProps } from 'components/image';
+import { Pillar } from 'pillar';
+
+
+// ----- Setup ----- //
+
+const sizes = `(min-width: ${breakpoints.phablet}px) 620px, 100vw`;
```

Comment:
Is the thumbnail `620px` wide from phablet and 100% of the viewport width otherwise?